docs/workflow/periodical-checks.md

```diff
@@ -1,95 +1,47 @@
-
-Document généré depuis les sources dans le dépôt git [@ORCA@gitremote@/tree/main/vault/docs](@ORCA@gitremote@/tree/main/vault/docs) au commit :  
+Document created from repository [@ORCA@gitremote@](@ORCA@gitremote@) at commit :  
 `@ORCA@commit@`
 
-## Protocole de vérification
+## Verifying this document
 
-> [!Tip]  
-> L'authenticité du contenu de ce document IN69 (au format HTML), doit être vérifié à l'aide de signature cryptographiques avant d'entamer son déroulement.
-> * Retrouver les identités publiques gpg des signataires (éditeur et vérificateur). Pour cela, se placer au commit signalé dans l'entête :
-> `git checkout @ORCA@commit@`
-> * À ce commit, les clés publiques se trouvent dans le répertoire [`src/workflow_signatory_keys/`](@ORCA@gitremote@/tree/main/src/workflow_signatory_keys).
-> * Vérifier que ces clés ont bien été insérées par un **commit signé par leur propriétaire**.
-> * Utiliser un nouveau trousseau gpg vide (toutes les commandes ci-dessous seront ensuite exécutées avec la variable d'environnement) :  
-> `export TMP_GPG_HOME=$(mktemp -d)`
-> * Importer toutes les clés publiques dans ce nouveau trousseau :  
-> `gpg --home="$TMP_GPG_HOME" --import /path/to/src/workflow_signatory_keys/*`
-> * Marquer toutes les clés comme de confiance ultime :  
-> `gpg --home="$TMP_GPG_HOME" --list-keys --keyid-format LONG --with-colons | sed -n -e '/^pub/{n;p}' | sed -n -E 's/^fpr:([^:]*:){8}([^:]*).*$/\2:6:/p' | gpg --home="$TMP_GPG_HOME"  --import-ownertrust`
-> * Extraire les signatures de la fin du document html :  
-> `sed -e '1,/^<!-- @GPG@SIGNATURES@ --><pre>$/ d' /chemin/vers/IN69_signed.html > /tmp/IN69.sig`
-> * Extraire la vestion html d'origine du document (sans signatures) :  
-> `sed -e '/^<!-- @GPG@SIGNATURES@ --><pre>$/q' /chemin/vers/IN69_signed.html > /tmp/IN69_without_signatures.html`
-> * Vérifier toutes les signatures :  
-> `gpg --home="$TMP_GPG_HOME" --verify /tmp/IN69.sig /tmp/IN69_without_signatures.html && echo "All signatures verified"`
-> * La validité des signatures est confirmée par l'affichage de la ligne `Good signature from xxxx` **autant de fois que de signataires du document** et qu'une dernière ligne `All signatures verified` s'affiche.
+Please follow your organisation's way of verifing a document to make sure this document has not been tempered with.
+A hand made one can be found at the [signing and verifying annex](../signing_and_verifying.md)
 
 ## Introduction
 
-Ce document décrit le déroulement de la cérémonie de vérifications récurrentes de la PKI d'Eove (chaîne de confiance basée sur des certificats digitaux).
-Ce processus implique l'ouverture de la PKI offline si des opérations doivent être effectuées sur cette dernière et dépend donc de l'exécution de la fiche IN65.
-
-## Historique des révisions
+This document explains how to periodically verify the PKI and its usability.
+This workflow may need to work on the offline PKI and this depends on the offline PKI workflow.
```

Comment:
Executing this workflow may require performing changes on the offline PKI, and thus executing an offline ceremony.

Reply:
see [02c0160](https://github.com/eove/orca/pull/2/commits/02c01609bb0f583e4ffe18a5f6f35876efcfb8b9)